keter-backend/src/main/java/ru/avplatonov/keter/keterbackend/initialize/Node.java

```diff
@@ -0,0 +1,95 @@
+package ru.avplatonov.keter.keterbackend.initialize;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Node {
+    private int id = 0;
+    private String name = "KETER";
+    private String description = "KETER";
+    private List<String> tags = null;
+    private String script = "KETER";
+    private String parameters = "KETER";
+    private List<List<String>> hardware = null;
```

Comment:
hardware is a Map<HardwareType, Long> where HardwareType - enum with type of hardware requirement and Long - value of needed resource